yt_dlp/extractor/ard.py

```diff
@@ -360,11 +361,26 @@ def _extract_episode_info(self, title):
     def _real_extract(self, url):
         display_id = self._match_id(url)
 
+        query = {
+            'embedded': 'false',
+            'mcV6': 'true'
+        }
+        headers = {}
+
+        token = self._download_json('https://sso.ardmediathek.de/sso/token', display_id, fatal=False)
```

Comment:
can we check for cookies before making this request? e.g.

```py
        if self._get_cookies(url).get('COOKIENAME'):
            token = self._download_json(
                'https://sso.ardmediathek.de/sso/token', display_id, fatal=False)
```
of course replacing `COOKIENAME` with an actually relevant cookie name

this would also prevent an unnecessary warning below

Reply:
Yeah good idea - we can check for ams 

I also considered checking if the user passed any cookies - without them we always do no authentication and if cookies are passed we check and warn?

background it that it is hard to see if the cookie is there so the user should get some feedback about it

but maybe not getting any feedback is also something ~

Reply:
Oh actually I can just adjust the warning for FSK Blocked to inform the user that they can authenticate with cookies

Reply:
Done 😄